test/scripts/find_pkg.js

```diff
@@ -1,66 +1,73 @@
 'use strict';
 
-const should = require('chai').should();
-const fs = require('hexo-fs');
-const pathFn = require('path');
+const { should } = require('chai');
```

Comment:
```suggestion
const { should } = require('chai');
require('chai').should();
```

Is the `should` initialization required?

Reply:
I'm debugging this